app/src/pages/inside/testCaseLibraryPage/duplicateTestCaseModal/duplicateTestCaseModal.scss

```
@@ -0,0 +1,46 @@
+  line-height: $line-height;
+  text-indent: 1px;
+  letter-spacing: 0;
+  color: var(--rp-ui-base-e-300);
+}
+
+.folder-select {
+  margin-top: 16px;
+
+  input::placeholder {
+    @include input-placeholder(28px);
+    vertical-align: middle;
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)